modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -1025,6 +1025,34 @@ func (suite *KeeperTestSuite) TestQueryPacketAcknowledgements() {
 			},
 			true,
 		},
+		{
+			"success, filtered res",
+			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				suite.coordinator.Setup(path)
+
+				for i := uint64(0); i < 10; i++ {
```

Comment:
It might be good to make the amount of acknowledgements in state large and the pagination limit small. This way we can test that we are only querying for acks for the commitments provided. I would try adjusting this so that the current code fails, then fix the code